dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -59,10 +59,10 @@
 
             <StaticResource x:Key="NavigationViewSelectionIndicatorForeground" ResourceKey="SystemControlForegroundAccentBrush" />
 
-            <StaticResource x:Key="TopNavigationViewItemForeground" ResourceKey="SystemControlForegroundBaseMediumBrush" />
-            <StaticResource x:Key="TopNavigationViewItemForegroundPointerOver" ResourceKey="SystemControlForegroundBaseMediumHighBrush" />
-            <StaticResource x:Key="TopNavigationViewItemForegroundPressed" ResourceKey="SystemControlForegroundBaseMediumHighBrush" />
-            <StaticResource x:Key="TopNavigationViewItemForegroundSelected" ResourceKey="SystemControlForegroundBaseHighBrush" />
+            <StaticResource x:Key="TopNavigationViewItemForeground" ResourceKey="SystemControlHighlightAltBaseHighBrush" />
```

Comment:
Since they map to the same hex codes, I think it's fine. If this PR would still be open, I would change it, but creating a PR just to change brushes so we reference the a different SystemBrush for consistency, despite that resulting in no change of color seems a bit overkill. 

If you like, you can create a PR for this, I would be more than happy to review that and accept it :)

Reply:
I wish I could've commented earlier but I don't really notice and study the changes until they are merged and ready to be ported to my WPF library 😅. Let's wait for the team's opinion on https://github.com/microsoft/microsoft-ui-xaml/pull/2113#discussion_r407260246 before proceeding.